agent/src/com/cloud/agent/IAgentShell.java

```diff
@@ -22,33 +22,64 @@
 import com.cloud.utils.backoff.BackoffAlgorithm;
 
 public interface IAgentShell {
-    public Map<String, Object> getCmdLineProperties();
+    Map<String, Object> getCmdLineProperties();
 
-    public Properties getProperties();
+    Properties getProperties();
 
-    public String getPersistentProperty(String prefix, String name);
+    String getPersistentProperty(String prefix, String name);
 
-    public void setPersistentProperty(String prefix, String name, String value);
+    void setPersistentProperty(String prefix, String name, String value);
 
-    public String getHost();
+    String getNextHost();
 
-    public String getPrivateIp();
+    String getPrivateIp();
 
-    public int getPort();
+    int getPort();
 
-    public int getWorkers();
+    int getWorkers();
 
-    public int getProxyPort();
+    int getProxyPort();
 
-    public String getGuid();
+    String getGuid();
 
-    public String getZone();
+    String getZone();
 
-    public String getPod();
+    String getPod();
 
-    public BackoffAlgorithm getBackoffAlgorithm();
+    BackoffAlgorithm getBackoffAlgorithm();
 
-    public int getPingRetries();
+    int getPingRetries();
 
-    public String getVersion();
+    String getVersion();
+
+    void setHosts(String hosts);
+
+    /**
+     * Resets host counter to start from the beginning
+     */
+    void resetHostCounter();
```

Comment:
Remove comments, or add them for all the methods on this interface for consistency.

Reply:
I do not feel consistency is an issue here as much as the comment not giving more information than the methodname, so yes, please remove.